duneggd/SubDetector/TMS.py

```diff
@@ -24,91 +24,167 @@ def construct(self, geom):
+        #                                  0.5*(self.thinbox1Dimension[0]+self.thinbox2Dimension[0])+self.gapPosition[0],
+        #                                  Q("0m"),
+        #                                  Q("0m"))
         
-        rt_pos = geom.structure.Position( 'rtpos'+self.name,
-                                          -(0.5*(self.thinbox1Dimension[0]+self.thinbox2Dimension[0])+self.gapPosition[0]),
-                                          Q("0m"),
-                                          Q("0m"))
+        #rt_pos = geom.structure.Position( 'rtpos'+self.name,
+        #                                  -(0.5*(self.thinbox1Dimension[0]+self.thinbox2Dimension[0])+self.gapPosition[0]),
+        #                                  Q("0m"),
+        #                                  Q("0m"))
             
-        ctr_pos = geom.structure.Position( 'ctrpos'+self.name,
-                                           Q("0m"),
-                                           Q("0m"),
-                                           Q("0m"))
-
+        #ctr_pos = geom.structure.Position( 'ctrpos'+self.name,
+        #                                   Q("0m"),
+        #                                   Q("0m"),
+        #                                   Q("0m"))
+        steel_pos1 = geom.structure.Position( 'steel_pos1'+self.name,
+                                            -1.5*self.thinbox1Dimension[0]-1.5*self.gapPosition[0],
+                                            Q("0m"),
+                                            Q("0m"))
+        steel_pos2 = geom.structure.Position( 'steel_pos2'+self.name,
+                                            -0.5*self.thinbox1Dimension[0]-0.5*self.gapPosition[0],
+                                            Q("0m"),
+                                            Q("0m"))
+        steel_pos3 = geom.structure.Position( 'steel_pos3'+self.name,
+                                            0.5*self.thinbox1Dimension[0]+0.5*self.gapPosition[0],
+                                            Q("0m"),
+                                            Q("0m"))
+        steel_pos4 = geom.structure.Position( 'steel_pos4'+self.name,
+                                            1.5*self.thinbox1Dimension[0]+1.5*self.gapPosition[0],
+                                            Q("0m"),
+                                            Q("0m"))
 
-        # Thin steel        
-        rt_pla = geom.structure.Placement( 'rtpla'+self.name, volume=thinBox1_lv, pos=rt_pos )
-        lf_pla = geom.structure.Placement( 'lfpla'+self.name, volume=thinBox1_lv, pos=lf_pos )
-        ctr_pla = geom.structure.Placement( 'ctrpla'+self.name, volume=thinBox2_lv, pos=ctr_pos )
 
-        thin_layer_lv.placements.append(rt_pla.name)
-        thin_layer_lv.placements.append(lf_pla.name)
-        thin_layer_lv.placements.append(ctr_pla.name)
+        # Thin steel
+        """Same here"""
```

Comment:
"Same here" is not very descriptive. The comment should stand on its own. Also it would be better if the alternative geometries would be continuous blocks that can be commented in and out, rather than interleaving things. Makes it easier and less prone to mistakes when switching.